domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/datamapper/dto/AksjonspunktDtoMapper.java

```diff
@@ -1,27 +1,36 @@
 package no.nav.foreldrepenger.melding.datamapper.dto;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
 
+import no.nav.foreldrepenger.fpsak.dto.behandling.aksjonspunkt.AksjonspunktDto;
 import no.nav.foreldrepenger.fpsak.dto.behandling.aksjonspunkt.AksjonspunkterDto;
 import no.nav.foreldrepenger.melding.aksjonspunkt.Aksjonspunkt;
-import no.nav.foreldrepenger.melding.kodeverk.KodeverkRepository;
+import no.nav.foreldrepenger.melding.aksjonspunkt.AksjonspunktDefinisjon;
 
 @ApplicationScoped
 public class AksjonspunktDtoMapper {
-    private KodeverkRepository kodeverkRepository;
 
-    public AksjonspunktDtoMapper() {
-    }
+    public List<Aksjonspunkt> mapAksjonspunktFraDto(AksjonspunkterDto dto) {
+        List<Aksjonspunkt> aksjonspunktList = new ArrayList<>();
+        for (AksjonspunktDto aksjonspunktDto : dto.getAksjonspunktDtos()) {
+            aksjonspunktList.add(Aksjonspunkt.ny()
+                    .medAksjonspunktDefinisjon(finnAksjonspunktDefinisjon(aksjonspunktDto.getDefinisjon().getKode()))
+                    .build());
+        }
 
-    @Inject
-    public AksjonspunktDtoMapper(KodeverkRepository kodeverkRepository) {
-        this.kodeverkRepository = kodeverkRepository;
+        return aksjonspunktList;
     }
 
-    public List<Aksjonspunkt> mapAksjonspunktFraDto(AksjonspunkterDto dto) {
-        return null;
+    private AksjonspunktDefinisjon finnAksjonspunktDefinisjon(String kode) {
+        if (AksjonspunktDefinisjon.VARSEL_REVURDERING_ETTERKONTROLL.getKode().equals(kode)) {
+            return AksjonspunktDefinisjon.VARSEL_REVURDERING_ETTERKONTROLL;
+        } else if (AksjonspunktDefinisjon.VARSEL_REVURDERING_MANUELL.getKode().equals(kode)) {
```

Comment:
deette skalerer ikke så bra da :P

Reply:
vi trenger ikke mer enn de to verdiene. Hvis vi trenger alle kodeverk da kan vi bruk Enum.valueOf